### PR TITLE
Adding documentation for new sensors on myStrom switches

### PR DESCRIPTION
**Pull request by jdebetaz**
## Proposed change

Informs people that two new sensors are available for myStrom switches

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/97024
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/mystrom.markdown [hunk 1]
```diff
@@ -29,12 +29,18 @@ There is currently support for the following device types within Home Assistant:
...
- - Switch Zero (120)
+
+- Switch CH v1 (101)
+- Bulb (102)
+- LED strip (105)
+- Switch CH v2 (106)
+- Switch EU (107)
+- Switch Zero (120)
+
+Two new sensors are available for switches:
```
Comment (fabaff) [inline source/_integrations/mystrom.markdown]: The consumption was always available not as a sensor but an attribute. We should simply mention what available.

```suggestion
Two sensors are available for switches:
```

**Comment by jdebetaz** [comment]
@fabaff would it be possible for you to review the request again since the parent one has been merged?

File: source/_integrations/mystrom.markdown [hunk 2]
```diff
@@ -29,12 +29,18 @@ There is currently support for the following device types within Home Assistant:
...
+- Bulb (102)
+- LED strip (105)
+- Switch CH v2 (106)
+- Switch EU (107)
+- Switch Zero (120)
+
+Two sensors are available for switches:
+
+- temperature
+- energy consumption
```
Comment (frenck) [inline source/_integrations/mystrom.markdown]:
```suggestion
- Temperature
- Energy consumption
```

**Review by frenck — APPROVED** [review]
Thanks, @jdebetaz 👍 

../Frenck

**Pull request merged**